### PR TITLE
Fixed Alignment problem and Indentation

### LINKED ISSUE #141: Alignment and Link problem in website.
* There is a very weird alignment problem on the website.
* The social links don't seem to be in place.

*It starts after the Topics section*

**I am working on this. I will have a Pull Request ready soon. So don't start to work on this.**

### PR DESCRIPTION
**Pull request by nzec**
* Fixed Alignment Problem.
* Fixed Indentation.
* Fixed #141 

***Preview Link : https://duttaditya18.github.io/2016.fossasia.org/***

---

### CODE REVIEW

**Review by anshumanv — CHANGES_REQUESTED** [review]
Many of your changes are not indented correctly, please fix.

**Review by NaviRocker — CHANGES_REQUESTED** [review]
Try to Fix Indentation.

**Comment by nzec** [comment]
>Many of your changes are not indented correctly, please fix.

@anshumanv @naveentherocker I fixed the indentation of almost the whole `index.html`.
Spent quite a bit of time on it! Please check it out!

**Review by samyakbvs — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -467,14 +442,13 @@ <h2>Cat Allman</h2>
 											<div class="social-links vertical-align">
 
 												<a href="https://fr.linkedin.com/in/davidestorti" target="default"><i class="icon social_linkedin"></i></a>
-                                                                                                <a href="https://www.facebook.com/davide.storti.73"><i class="icon social_facebook"></i></a>
-                                                                                                <a href="https://www.instagram.com/davidone_91/?hl=en"><i class="icon social_instagram"></i></a>
-                                                                                                <a href="https://twitter.com/dstorti21?lang=en"><i class="icon social_twitter"></i></a>
+                                                <a href="https://www.facebook.com/davide.storti.73"><i class="icon social_facebook"></i></a>
```
Comment (anshumanv) [inline index.html]: Still not indented correctly.

File: index.html [hunk 2]
```diff
@@ -493,15 +467,14 @@ <h2>Davide Storti</h2>
...
 												<a target="default" href="https://twitter.com/MikeMcQuaid" target="default"><i class="icon social_twitter"></i></a>
 												<a target="default" href="https://uk.linkedin.com/in/mkmcqd" target="default"><i class="icon social_linkedin"></i></a>
-                                                                                                <a target="default" href="https://www.facebook.com/mkmcqd" target="default"><i class="icon social_facebook"></i></a>
-                                                                                                <a target="default" href="https://github.com/MikeMcQuaid" target="default"><i class="icon social_github"></i></a>
-                                                                                                <a target="default" href="https://www.instagram.com/mcquaid.mike/?hl=en"><i class="icon social_instagram"></i></a>
-												<a target="default" href="https://mikemcquaid.com/"><i class="icon fa fa-external-link"><i></a>	
+	                                            <a target="default" href="https://www.facebook.com/mkmcqd" target="default"><i class="icon social_facebook"></i></a>
```
Comment (anshumanv) [inline index.html]: same

File: index.html [hunk 3]
```diff
@@ -520,14 +493,13 @@ <h2>Mike McQuaid</h2>
 										<div class="hover-state text-center preserve3d">
 											<div class="social-links vertical-align">
 												<a href="https://twitter.com/bytebot" target="default"><i class="icon social_twitter"></i></a>
-                        									<a href="https://www.instagram.com/bytebot/"><i class="icon social_instagram"></i></a>
+            									<a href="https://www.instagram.com/bytebot/"><i class="icon social_instagram"></i></a>
```
Comment (anshumanv) [inline index.html]: same

**Review by anshumanv — COMMENTED** [review]
I've pointed out some more, there are others too, please see diff.

**Comment by nzec** [comment]
@anshumanv Fixed!

**Comment by nzec** [comment]
Ping @anshumanv

**Review by anshumanv — APPROVED** [review]

**Review by NaviRocker — APPROVED** [review]
LGTM

**Review by Ploypiti — APPROVED** [review]

**Review by saurabhjn76 — APPROVED** [review]

**Comment by nzec** [comment]
@saurabhjn76 Thanks!

**Pull request merged**